pyfr/solvers/euler/kernels/entropy.mako

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+<%inherit file='base'/>
+<%namespace module='pyfr.backends.base.makoutil' name='pyfr'/>
+
+<% inf = 1e20%>
+<%pyfr:macro name='compute_entropy' params='u, d, p, e'>
+    d = u[0];
+    fpdtype_t E = u[${nvars - 1}];
+
+    // Compute the velocities
```

Comment:
Not sure here? You're extracting the momenta. Not sure if you ever need to define rhov, can just do the dot product directly.